### PR TITLE
Add MCNC 2020 article

### PR DESCRIPTION
**Pull request by xz1z1z1z1z**
Add MCNC 2020 article and fix some faults in 2019 article

### CODE REVIEW

File: wiki/Tournaments/MCNC/2019/en.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ This competition has come to an end and resulted in the following podium:
 
 **[Download the mappack here! (110MB)](https://www.dropbox.com/s/6ocxk2bm9w8q49j/MCNC19%20%E5%9B%9B%E5%88%86%E4%B9%8B%E4%B8%80%E5%86%B3%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
 
-- ModType
+- Freemod
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2019/en.md:156]:
```suggestion
- FreeMod
```

File: wiki/Tournaments/MCNC/2019/en.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ This competition has come to an end and resulted in the following podium:
 
 **[Download the mappack here! (82.9MB)](https://www.dropbox.com/s/k752wba6kpyyzt1/MCNC19%20RO16%E5%9B%BE%E5%8C%85.rar?dl=0)**
 
-- ModType
+- Freemod
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2019/en.md:177]:
```suggestion
- FreeMod
```

File: wiki/Tournaments/MCNC/2020/en.md [hunk 3]
```diff
@@ -0,0 +1,406 @@
+---
+tags:
+  - MCNC
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2020/en.md:3]:
```suggestion
  - MCNC 2020
```

File: wiki/Tournaments/MCNC/2020/en.md [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+  - MCNC2020
+---
+
+# osu!mania 4K Chinese National Cup 2020
+
+![MCNC 2020 logo](img/logo.jpg)
+
+The **osu!mania 4K Chinese National Cup 2020** (***MCNC 2020***) was a country-based osu!mania tournament hosted by the \[Crz\]Team. It was the second installment of the osu!mania 4K Chinese National Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2020/en.md:11]:
```suggestion
The **osu!mania 4K Chinese National Cup 2020** (***MCNC 2020***) was a country-based osu!mania tournament hosted by the ![][flag_CN] \[Crz\]Team. It was the second installment of the osu!mania 4K Chinese National Cup.
```

File: wiki/Tournaments/MCNC/2020/en.md [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+
+## Mappools
+
+### Finals
+
+**[Download the mappack here! (201MB)](https://www.dropbox.com/s/p4daduyfm4us6m4/MCNC20%20%E5%86%B3%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- Freemod
+  - [Aquellex - Tachyon Beam Cannon (Plutes) Gekido-'s \[Destruction 1.1x\]](https://osu.ppy.sh/b/2476292)
+  - [Camellia - * Erm, could it be a Spatiotemporal ShockWAVE Syndrome…? (Sillyp) \[NIGHTMARE\]](https://osu.ppy.sh/b/2483411)
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2020/en.md:126]:
```suggestion
  - [Camellia - \* Erm, could it be a Spatiotemporal ShockWAVE Syndrome…? (Sillyp) \[NIGHTMARE\]](https://osu.ppy.sh/b/2483411)
```

File: wiki/Tournaments/MCNC/2020/en.md [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_CN] [Kakuko](https://osu.ppy.sh/users/10413362) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_CN] [\[Crz\]I-Night](https://osu.ppy.sh/users/8790619) |
+
+![MCNC 2020 podium](img/podium.png)
+![MCNC 2020 bracket](img/bracket.png)
+
+## Mappools
+
+### Finals
+
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2020/en.md:121]: `<pack size>MB` -> `<pack size> MB`

File: wiki/Tournaments/MCNC/2020/en.md [hunk 7]
```diff
@@ -0,0 +1,406 @@
...
+| **\[Crz\]Willis** | **5** | 0 | - xiaoluoli - | [#1](https://osu.ppy.sh/community/matches/61244894) |
+| **\[Crz\]I-Night** | **5** | 0 | Kagaku | [#1](https://osu.ppy.sh/community/matches/61238984) |
+| **ImmortalphoeniX** | **5** | 1 | GODV14138 | [#1](https://osu.ppy.sh/community/matches/61236550) |
+| **CaOH2** | **5** | 0 | \[ Emiya \] | [#1](https://osu.ppy.sh/community/matches/61209240) |
+| **\[Crz\]FolAH1217** | **5** | 0 | Cloud18 | [#1](https://osu.ppy.sh/community/matches/61206938) |
+| **\[RT\]Alleyne** | **5** | 0 | \[ Emiya \] | *Win by Default* |
+| **Hx\_032** | **5** | 0 | Kagaku | [#1](https://osu.ppy.sh/community/matches/61201441) |
+| **\[RT\]Alleyne** | **5** | 1 | CherryG | [#1](https://osu.ppy.sh/community/matches/61200581) |
+
+
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2020/en.md:343]: delete those 2 lines

File: wiki/Tournaments/MCNC/2020/en.md (hunk 3 above)
Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2020/en.md:3]: MCNC 2020 is in Line 4

File: wiki/Tournaments/MCNC/2020/en.md (hunk 4 above)
Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2020/en.md:11]: I think this is not necessary

File: wiki/Tournaments/MCNC/2019/en.md (hunk 1 above)
Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2019/en.md]: Done

File: wiki/Tournaments/MCNC/2020/en.md (hunk 5 above)
Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2020/en.md]: Done

File: wiki/Tournaments/MCNC/2020/en.md (hunk 6 above)
Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2020/en.md:121]: Done

File: wiki/Tournaments/MCNC/2020/en.md (hunk 7 above)
Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2020/en.md]: Done

File: wiki/Tournaments/MCNC/2020/en.md (hunk 3 above)
Comment (Pennek) [reply wiki/Tournaments/MCNC/2020/en.md:3]: tags are there to help the search engine categorise better. it values tags much higher than text so adding this is worthwhile

Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2020/en.md:3]: I mean, there are 2 tags in the article, MCNC and MCNC 2020. The tag MCNC 2020 is in Line 4

Comment (TPGPL) [reply wiki/Tournaments/MCNC/2020/en.md:3]: `MCNC2020` and `MCNC 2020` are two different tags

Comment (cl8n) [reply wiki/Tournaments/MCNC/2020/en.md]: the tags here should be `MCNC`, and `MCNC2020`. a search for "MCNC 2020" will match the MCNC tag and 2020 from the title, you don't need that in tags

File: wiki/Tournaments/MCNC/2020/en.md [hunk 8]
```diff
@@ -0,0 +1,404 @@
...
+  - [Hong Liu Zhi Yuan - Mouse Wizard (\[RT\]Alleyne) \[Mouse\]](https://osu.ppy.sh/b/2126573)
+  - [Hommarju - BEAST BASS BOMB (-MysticEyes) \[MAXIMUM\]](https://osu.ppy.sh/b/1940816)
+  - [Nanawo Akari - DaDaDaDa Tenshi (Xingyue-) \[Dame\]](https://osu.ppy.sh/b/2352772)
+  - [ITSUHA. - Klouz of Reveria (Elekton) \[cascade\]](https://osu.ppy.sh/b/1514656)
+- Tiebreaker
+  - **[Camellia - Tornado (Dubstek) \[Disaster\]](https://osu.ppy.sh/b/2021759)**
+
+### Qualifiers
+
+**[Download the mappack here! (14.3 MB)](https://www.dropbox.com/s/xwvugt6bvhy6d20/MCNC20%20%E8%B5%84%E6%A0%BC%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2020/en.md:229]:
```suggestion
**[Download the mappack here! (14 MB)](https://www.dropbox.com/s/xwvugt6bvhy6d20/MCNC20%20%E8%B5%84%E6%A0%BC%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
```

File: wiki/Tournaments/MCNC/2020/en.md [hunk 9]
```diff
@@ -0,0 +1,404 @@
...
+
+- NoMod
+  - [DASU - 8 6 (\[Crz\]xz1z1z) \[SV\]](https://osu.ppy.sh/b/2285630)
+  - [Falk - Mach Sigma - For Dream Dance Act 1 (shuniki) \[ShuChan!!\]](https://osu.ppy.sh/b/1996937)
+  - [AKINO from bless4 - Miiro (Mat) \[Acc Meme\]](https://osu.ppy.sh/b/1759637)
+  - [KBSM - Blue Rose (LegendXYZ) \[Hard\]](https://osu.ppy.sh/b/237790)
+  
+## Match results
+
```
Comment (TPGPL) [inline wiki/Tournaments/MCNC/2020/en.md:238]: `*Win by Default*` -> `*win by default*`

Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2020/en.md:238]: *win by default* changed
but i dont think we need to use 0 | -1 when MWC use MAX | 0

**Pull request merged**